### ISSUE TITLE
[Bug]: A JavaScript error occured in the main process

### ISSUE THREAD

**Issue by Nidzo22** [ThaUnknown/miru#185]
### Preflight checklist

- [X] I have searched the [issue tracker](https://github.com/ThaUnknown/miru/issues) for a bug report that matches the one I want to file, without success.
- [X] I have searched the [frequently asked questions](https://github.com/ThaUnknown/miru/blob/master/docs/faq.md) for a solution that fixes this problem, without success.
- [X] I have checked that I'm using the [latest stable](https://github.com/ThaUnknown/miru/releases/latest) version of the app.

### What app version are you using?

v2.4.3

### What operating system version are you using?

Windows

### Expected Behavior

The app should open normally (it was fine earlier today).

### Actual Behavior

This is the error message that shows up when I run the app (admin and non-admin).

![123123](https://user-images.githubusercontent.com/20055775/172071758-a9d0ed23-4f93-46c3-b95f-378fd8c351a4.PNG)

**Comment by ThaUnknown** [comment]
major panik

**Comment by ThaUnknown** [comment]
https://github.com/ThaUnknown/miru/commit/450c6d6311b421b7039115fac1c59229cd9e7496

**Comment by ThaUnknown** [comment]
I uuuuh.... fucked it, you'll need to manually re-install a new version. you can download it from: https://github.com/ThaUnknown/miru/releases/tag/v2.4.5

**Comment by ThaUnknown** [comment]
actually, gonna leave this open for a few days in case people come here to check

**Comment by Nidzo22** [comment]
Lol I ended up just getting builder-until-runtime on my own and then unpacked the app.asar and repacked it back to .asar

**Comment by ThaUnknown** [comment]
> Lol I ended up just getting builder-until-runtime on my own and then unpacked the app.asar and repacked it back to .asar

ahaha, I wouldn't even know how to do that, you should help me maintain this dude XD

TLDR the issue is I wanted to use PNPM which is like x2 faster on windows to install deps, but that fucked something so I reverted it

**Comment by Nidzo22** [comment]
> > Lol I ended up just getting builder-until-runtime on my own and then unpacked the app.asar and repacked it back to .asar
> 
> ahaha, I wouldn't even know how to do that, you should help me maintain this dude XD
> 
> TLDR the issue is I wanted to use PNPM which is like x2 faster on windows to install deps, but that fucked something so I reverted it

oh lol, as soon as I discovered the issue I was waiting for an update cuz i was assuming that the file got fucked or something. In the error it says that it cant find builder-util-runtime and i was trying for the last couple of hours to install it in the command line, eventually it worked but I had to repack it back into the .asar file. I dont mess around with the command line a lot so this was an interesting first for me lol. Also as soon as I figured it out myself I see that you already fixed the issue and you updated it lol
